tests/publiccloud/download_repos.pm

```diff
@@ -111,9 +111,6 @@ sub run {
     record_info("Repo size", "Total repositories size: $total_size");
     assert_script_run("find ./ -name '*.rpm' -exec du -h '{}' + | sort -h > /root/rpm_list.txt", timeout => 60);
     upload_logs("/root/rpm_list.txt");
-
-    # The maintenance *.repo files all point to download.suse.de, but we are using dist.suse.de, so we need to rename the directory
-    assert_script_run("if [ -d ~/repos/dist.suse.de ]; then mv ~/repos/dist.suse.de ~/repos/download.suse.de; fi");
```

Comment:
what if instead:
```
assert_script_run("if [ -d ~/repos/dist.suse.de ]; then mv ~/repos/dist.suse.de ~/repos/download.suse.de; fi") unless (get
_var('REPO_MIROR_HOST') eq "dist.suse.de");
````

Reply:
that is, let it still rename folder when MIRROR is Not dist.

Reply:
I didn't intend to add more options, but to prevent presence of `dist.suse.de` subdirectory in the case of _mistakes_, when nowhere it is referenced. But considering a future update _without any url reference_, it is ok to remove it now.